warehouse/forklift/legacy.py

```diff
@@ -154,6 +156,8 @@ def _valid_platform_tag(platform_tag):
     if platform_tag in _allowed_platforms:
         return True
     m = _macosx_platform_re.match(platform_tag)
+    if m and int(m.group("major")) < 10:
```

Comment:
If it's preferable to make new mac major version support opt-in, I'll switch this to a `_macosx_major_versions` allowed-set like `_macosx_arches`

Reply:
Yes, let's do that.

Reply:
Sure thing! Done.